crates/forge/tests/it/zk/nft.rs

```diff
@@ -0,0 +1,59 @@
+use foundry_test_utils::{forgetest_async, util, TestCommand, TestProject, ZkSyncNode};
+
+forgetest_async!(script_zk_can_deploy_nft, |prj, cmd| {
+    setup_nft_prj(&mut prj);
+    run_nft_script_test(prj.root(), &mut cmd, "MyScript", 1);
```

Comment:
I had this pattern with the Factory scripts simply because I had multiple tests - I was gonna suggest you to inline these, but it looks like this is a recurring pattern so perhaps `run_nft_script_test` (and `run_factory_script_test`) could be rewritten in a more general way